### PR TITLE
Access controller: ownership transfer

### PR DESCRIPTION
**Pull request by archseer**

### CODE REVIEW

File: contracts/programs/access-controller/src/lib.rs [hunk 1]
```diff
@@ -19,8 +19,8 @@ pub const MAX_ADDRS: usize = 64;
 
 #[zero_copy]
 pub struct AccessList {
-    xs: [Pubkey; 64], // sadly we can't use const https://github.com/project-serum/anchor/issues/632
     len: u64,
+    xs: [Pubkey; MAX_ADDRS],
```
Comment (krebernisak) [inline contracts/programs/access-controller/src/lib.rs:23]: Why the order switch here?

I've noticed you usually put data first, and the data length last.

Comment (krebernisak) [reply contracts/programs/access-controller/src/lib.rs:23]: https://github.com/smartcontractkit/chainlink-solana/runs/4786341439?check_suite_focus=true

**Comment by aalu1418** [comment]
tested locally with ecr build: develop.f20690e8ede0cfead9df7f808f56a14f26469aaa

**Review by aalu1418 — APPROVED** [review]
🚀

**Pull request merged**